code/game/machinery/portable_turret.dm

```diff
@@ -208,6 +208,8 @@
 	check_all = TRUE
 	can_salvage = FALSE	// So you can't just twoshot a turret and get a fancy gun
 
+// Start Lasertag Turret Code

```

Comment:
This can probably be removed, since this is a polaris file, any changes would need to be marked with //vorestation edit and there is no reason to have a comment

Reply:
Also fixed